resources.qrc

```diff
@@ -5,6 +5,7 @@
         <file>ts/2048-qt_pl_PL.qm</file>
         <file>ts/2048-qt_ru_RU.qm</file>
         <file>ts/2048-qt_zh_CN.qm</file>
+        <file>ts/2048-qt_es_ES.qm</file>
```

Comment:
wrongly placed, please keep it sorted alpabetically -> put this to line 4 (after ` <file>ts/2048-qt_de_DE.qm</file>`)